roles/kubernetes-apps/csi_driver/vsphere/templates/vsphere-csi-controller-config.yml.j2

```diff
@@ -1,28 +1,29 @@
+{%- set default_config = {
```

Comment:
Instead of doing this in jinja, I'd just add two ansible variable like
vsphere_config_base == this stuff
vsphere_config_override: {}

And do something like `vsphere_config_base | combine(vs_c_override) | to_yaml`